### DISCUSSION TITLE
White bar above Firefox menu icon

### DISCUSSION THREAD

**Discussion by eagrunewald** [Godiesc/firefox-gx#168] [Q&A] [answered]
Hi, after Firefox was updated, it shows a white bar above the Firefox menu (top left corner). It doesn't appear when I open a private window. Does anyone know how to fix this?

![firefox_gx](https://github.com/Godiesc/firefox-gx/assets/61214231/769470bd-c697-4948-b0a2-9f96bd8df36b)

**Comment by Godiesc** [comment, marked as answer]
hi, it seems like you are using an old version of the theme, were you try to update the theme (same steps like the first time you install the theme)?

**Comment by eagrunewald** [comment]
You're right, I must have messed something up here. I tried installing from scratch and it worked fine, thanks!
